CONTRIBUTING.md

```diff
@@ -121,13 +97,35 @@ The checks are configured to run on the `devel` branch of each external reposito
 
 Maintainers will gladly help you in this process.
 
-## Final Approval and Submission
+### Final Approval and Submission
 
 Once all corrections have been made, automated checks are passing, and a maintainer has given final approval, it is time
 to contribute the submission. A maintainer will handle this final step and once complete changes should appear in the
 `devel` branch. You can help this process by submitting any deferred or future work items as issues using the links
 above.
 
+## Project Structure
+
+The F´ project is designed as a base software [framework](https://github.com/nasa/fprime) with additional
+[packages](https://github.com/fprime-community) designed to extend the framework. This means that occasionally we may
+move contributions in or out of these packages.
+
+Key packages include:
+
+- [fpp](https://github.com/fprime-community/fpp): fpp development repository
+- [fprime-tools](https://github.com/fprime-community/fprime-tools): `fprime-util` development repository
+- [fprime-gds](https://github.com/fprime-community/fprime-gds): `fprime-gds` development repository
+
+
+### F´ Repository Structure
+
+Contributors to the [fprime](https://github.com/nasa/fprime) repository should understand the following key folders:
+
+- [docs/UsersGuide](https://github.com/nasa/fprime/tree/devel/docs/UsersGuide): add new documentation in this or a subfolder
+- [Fw](https://github.com/nasa/fprime/tree/devel/Fw): changes here will be reviewed carefully because this code is critical across F
+- [Ref](https://github.com/nasa/fprime/tree/devel/Ref): update and maintain the Reference application here
```

Comment:
These paths do not exist anymore, I recommend pointing to the "A Tour of the Source Tree" page:

```suggestion
Contributors to the [fprime](https://github.com/nasa/fprime) repository should refer to the following guide to understand the repository structure: [A Tour of the Source Tree](docs/user-manual/overview/source-tree.md)
```